dataland-backend/src/main/resources/application-nodb.properties

```diff
@@ -9,4 +9,5 @@ dataland.apikeymanager.base-url=https://local-dev.dataland.com/api-keys
 dataland.internalstorage.base-url=https://local-dev.dataland.com/internal-storage
 spring.rabbitmq.listener.simple.auto-startup=false
 spring.flyway.enabled=false
-spring.servlet.multipart.max-file-size=2MB
+spring.servlet.multipart.max-file-size=100MB
```

Comment:
Reicht das aus? Wir haben doch teilweise schon PDFs die größer sind als das.

Reply:
Das hier ist das nodb Profil für Entwicklung.  Check mal ob der Wert in den prod properties ok ist.